src/Apps/Auction/Components/BidForm.tsx

```diff
@@ -11,14 +11,15 @@ import { BidForm_saleArtwork } from "__generated__/BidForm_saleArtwork.graphql"
 import { PricingTransparency } from "Apps/Auction/Components/PricingTransparency"
 import { ConditionsOfSaleCheckbox } from "Components/Auction/ConditionsOfSaleCheckbox"
 import { Form, Formik, FormikActions, FormikValues } from "formik"
-import { dropWhile } from "lodash"
+import { dropWhile, findLast } from "lodash"
```

Comment:
should probably change this to `find`, was previously selecting the last increment below the amount so 501 would fall back to the 500 increment.